axon-server/performance/flow-control.md

```diff
@@ -29,5 +29,23 @@ Set the following properties to set flow control on the synchronization between
 * `axoniq.axonserver.queryFlowControl.nr-of-new-permits` \[5000\] - additional number of messages that the master can send to replica.
 * `axoniq.axonserver.queryFlowControl.new-permits-threshold` \[5000\] - when replica reaches this threshold in remaining messages, it sends a request with additional number of messages to receive.
 
+## Streaming query
+
+Flow control and stream cancellation features are only available with Axon Server 4.6 + version. Streaming queries used with pre Axon Server 4.6 version will work, but without these important features.
+Under the hood, backpressure does `Hop to Hop` signal propagation (see below) and inherits gRPC's [HTTP2-based backpressure model](https://developers.google.com/web/fundamentals/performance/http2/#flow_control).
```

Comment:
```suggestion
Flow control and stream cancellation features are only available with Axon Server 4.6.0 and up. 
When streaming queries are used with Axon Server versions before 4.6.0, it will work although without the following essential features.
Under the hood, backpressure does `Hop to Hop` signal propagation (see below) and inherits gRPC's [HTTP2-based backpressure model](https://developers.google.com/web/fundamentals/performance/http2/#flow_control).
```